tools/avatarUploader.js

```diff
@@ -0,0 +1,64 @@
+/* flow */
+/* eslint-disable no-console, no-param-reassign */
+
+import { urlTos3 } from '../src/lib/upload';
+import { bus, config } from '../src/core-server';
+import { TYPE_USER } from '../src/lib/Constants';
+import buildAvatarURLForSize from '../src/lib/buildAvatarURLForSize';
+import promisify from '../src/lib/promisify';
+import winston from 'winston';
+import * as pg from '../src/lib/pg';
+
+type User = {
+	id: string;
+	meta: {
+		picture: string
+	};
+	params: {
+		picture: string
+	}
+};
+
+const IMAGE_NAME = 'avatar';
+const performReadQuery = promisify(pg.read.bind(pg, config.connStr));
+
+const uploadImageHelper = async (user: Array<User>, propName: string) => {
+	await urlTos3(buildAvatarURLForSize(user[propName].picture, 1024), 'a/' + user.id + '/' + IMAGE_NAME);
+	const changes = {
+		entities: {
+			[user.id]: {
+				type: TYPE_USER,
+				id: user.id,
+				[propName]: {
+					picture: null
```

Comment:
We use a library called [JsonOp](/belng/jsonop) to merge JSON blobs, which has a delete property operation (rather than setting to null). You can use this by doing `picture: ["$delete"]` instead of this.